build/ci/periodic_build.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2024 MongoDB Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+exec_timeout_secs: 4200 # automatically fail any task if it takes longer than 1h10m to finish.
+include:
+  - filename: build/ci/evergreen_common.yml
+  
+variables:
+  - &go_linux_version
+      go_root: "/opt/golang/go1.22"
+      go_bin: "/opt/golang/go1.22/bin"
+      go_base_path: ""
```

Comment:
question: can we have these vars in common? I see them in build/ci/evergreen.yml too

Reply:
unfortunately not the yaml anchor is per file